integration/test/definitions/custom-yarn-complex/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "custom-yarn-complex",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "engines": {
+    "yarn": ">=0.20.0 <0.21.0"
```

Comment:
it might we worth adding another test that installs a `dist-tag` (e.g. "latest" or "rc"). This can be done in a follow-on.

Reply:
That's a good idea.  I will add it in a follow-on.